notebooks/whisper-asr-genai/whisper-asr-genai.ipynb

```diff
@@ -75,8 +75,9 @@
     "\n",
     "\n",
     "%pip install -q \"torch>=2.3\" \"torchvision>=0.18.1\" --extra-index-url https://download.pytorch.org/whl/cpu\n",
-    "%pip install -q \"transformers>=4.45\" \"git+https://github.com/huggingface/optimum-intel.git\" --extra-index-url https://download.pytorch.org/whl/cpu\n",
-    "%pip install -q -U \"openvino>=2024.5.0\" \"openvino-tokenizers>=2024.5.0\" \"openvino-genai>=2024.5.0\"\n",
+    "%pip install -q -U \"transformers>=4.45\"\n",
```

Comment:
please add --extra-index-url

(transformers may install some torch specific packages and override user installed torch)

Reply:
added